app/cafe.py

```diff
@@ -0,0 +1,18 @@
+from app.errors import (
+    NotWearingMaskError, NotVaccinatedError, OutdatedVaccineError
+)
+import datetime
+
+
+class Cafe:
+    def __init__(self, name: str) -> None:
+        self.name = name
+
+    def visit_cafe(self, visitor: dict) -> str:
+        if "vaccine" not in visitor:
+            raise NotVaccinatedError
+        if datetime.date.today() > visitor["vaccine"]["expiration_date"]:
+            raise OutdatedVaccineError
```

Comment:
The error raised here lacks a descriptive message. According to the checklist, use descriptive error messages. For example: `raise OutdatedVaccineError('Visitor's vaccine is outdated.')`.